GitCommands/DiffListSortService.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitCommands
+{
+    public class DiffListSortService : IDiffListSortService
+    {
+        private static readonly Lazy<DiffListSortService> _lazyDiffListSorting = new Lazy<DiffListSortService>(() => new DiffListSortService());
+
+        public static DiffListSortService Instance => _lazyDiffListSorting.Value;
+
+        private DiffListSortType _diffListSorting;
```

Comment:
please consolidate all fields, properties and methods in their logical groups
this is a good reference - https://blogs.msdn.microsoft.com/brada/2005/01/26/internal-coding-guidelines/

Reply:
Are you just referring to the lines between the fields? Sorry, haven't gotten my new VS environment completely up to snuff on the new computer. Used to tools cleaning that up for me.